subsys/dfu/boot/mcuboot.h

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2017 Nordic Semiconductor ASA
+ * Copyright (c) 2016 Linaro Limited
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef __MCUBOOT_H__
+#define __MCUBOOT_H__
+
+#include "boot/mcuboot_constraints.h"
+
+/**
+ * @brief Marks the image in slot 0 as confirmed. The system will continue booting
+ * into the image in slot 0 until told to boot from a different slot.
```

Comment:
But, we need to be able to mark this for both slots, to be able to perform either upgrade type.